### PR TITLE
Made Live Search find input element and make it fetch course cards

### PR DESCRIPTION
**Pull request by ciss1995**
Made Live Search find input element and make it fetch course cards instead of listbox

### CODE REVIEW

File: src/search-filter/d2l-search-widget-custom.js [hunk 1]
```diff
@@ -325,54 +301,9 @@ Polymer({
...
-			var result = this.fetchSirenEntity(url)
-				.then(function(organizationEntity) {
-					// Polymer's splice method causes empty results to show up for an
-					// unknown reason, so use a normal Array.prototype.splice
-					this._liveSearchResults.splice(i, 0, {
-						name: organizationEntity.properties.name
-					});
-				}.bind(this));
-			fetches.push(result);
+			this._setSearchUrl(this._liveSearchPageSize, 'searchUrl');
```
Comment (ciss1995) [inline src/search-filter/d2l-search-widget-custom.js]: Fetch the cards directly instead of fetching the course name as suggestions in listbox

Comment (ciss1995) [reply src/search-filter/d2l-search-widget-custom.js]: Performance cost seems similar. I am trying to understand and run the Tracing GUI to track the number of db calls.

File: src/search-filter/d2l-search-widget-custom.js [hunk 2]
```diff
@@ -325,54 +301,9 @@ Polymer({
 
 	_liveSearch: function() {
 		if (this._searchInput.trim().length > 0) {
-			this._setSearchUrl(this._liveSearchPageSize, '_liveSearchUrl');
```
Comment (njostonehouse) [inline src/search-filter/d2l-search-widget-custom.js]: This still feels like unexpected behaviour to me, but it's not like we've changed it for the worse, I don't think. At worst, I'd say we could stand to keep the number of results at 5 if we were nervous.

File: src/search-filter/d2l-search-widget-custom.js [hunk 3]
```diff
@@ -325,54 +301,9 @@ Polymer({
...
-			if (req.abort) {
-				req.abort();
-			}
-		});
-		this.set('_organizationRequests', []);
-
-		return this.fetchSirenEntity(this._liveSearchUrl)
-			.then(this._onLiveSearchResponse.bind(this));
-	},
-	_onLiveSearchResponse: function(enrollmentsEntity) {
```
Comment (njostonehouse) [inline src/search-filter/d2l-search-widget-custom.js]: Goodbye live search suggestions! Don't let the door hit you on the way out!

**Pull request merged**